mmf/trainers/core/reporting.py

```diff
@@ -51,8 +51,8 @@ def update_dict(self, meter_update_dict, values_dict):
                 if val.dim() == 1:
                     val = val.mean()
 
-            if hasattr(val, "item"):
-                val = val.item()
+            #if hasattr(val, "item"):
```

Comment:
let's utilize the util function `item` as described in a previous comment instead of commenting out.